README.md

```diff
@@ -823,25 +845,6 @@ Run the tests:
 `pytest` -- This runs all the tests.
 `pytest test_gordon360_pytest.py -k '{name of def}'` -- This runs a specific test based on {name of def}.
 
-
-### Manual Testing
-
-#### Running the Server Locally
-
-* As you are probably using one of the Linux machines in the Computer Science lounge, you will need to be on the virtual machine to run the server locally. Follow the directions [here](RemoteDesktopToVM.md) to set up and connect to the virtual machine.
-
-* If this is your first time on the virtual machine, you will need to clone the 360 code. You can use something like Git Bash or VS Code to do this.
-
-* Before you open the gordon-360-api folder, you will have to add the `secrets.config` file to it. The file is located on the CS-RDSH-02 virtual machine in `C:\Users\Public\Public Documents\` (or `/c/users/public/documents\` when in git-bash). Copy the file `secrets.config` to the same folder in your project that contains the `web.config` file; currently, this is in `gordon-360-api\Gordon360`. This file is a sort of keyring for the server to authorize itself at various points.
-
-* Now, to open the api, look for the desktop app Visual Studio 2017, which has a purple Visual Studio icon. You will have to log in to a Microsoft account, which can just be the account Gordon made for you. Once you log in, go to `File > Open > Project/Solution`. Then, select and Open the file `gordon-360-api/Gordon360.sln`.
-
-* There is a little configuration you must yet do before running the server. In the solution explorer on the right, right click the name of the project (Gordon360) and select properties.  From the tabs on the left, choose the Web tab and change the Project Url to an unused port. For example, if you chose port 5555, change Project Url to `"http://localhost:5555"`. Then click Create Virtual Directory. Press OK on the dialog box, and you all configured!
```

Comment:
There should be an 'are' in 'you all configured' at the end of this line. That was my bad when I wrote this. . .

Reply:
Whoops, this was meant for 129, where this paragraph was moved to.

Reply:
Ah, makes sense. I can make the change in a future pull request

Reply:
Sounds good